includes/Core/Feature_Tours/Feature_Tours.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Feature_Tours\Feature_Tours
+ *
+ * @package   Google\Site_Kit\Core\Feature_Tours
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Feature_Tours;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Storage\User_Options;
+
+/**
+ * Class for handling feature tours.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Feature_Tours {
```

Comment:
This class seems a bit thin as it is, but I don't think that's necessarily a bad thing. I definitely prefer separating out the controller so this is just a small feature as it is 😄

Reply:
Yeah I like it, this class is basically the PHP entry point to the entire feature, so that `Plugin` has to only initialize that rather than every "sub-class" of the feature.